### ISSUE TITLE
2to3 doesn't work

### ISSUE THREAD

**Issue by FENG06** [ipython-contrib/jupyter_contrib_nbextensions#1298]
![image](https://upload-images.jianshu.io/upload_images/41125-572ea2c645a93604.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)

Hi.
I clone the git project and install correctly , but when i use the 2to3 converter,it can't work?

and the icon isn't the same style like the readme file.

Looking forward to your reply.

**Comment by jfbercher** [comment]
Indeed, there is a problem with the icon and config load which can be fixed. 
However the extension works well for me. 
Can you

1. put a simple python 2 instruction in a single cell, like 
```
print "This is a text"
```
2. Press the 2to3 button 

and report what happens. 
if nothing happens, look in the javascript console if there are error messages.

**Comment by FENG06** [comment]
@jfbercher Thanks for your reply. 

It does work when I try the simple print sentence.
```
print "This is a text"
```
But if I run the whole cell, it can't be converted. Anyway, I will check my code again.
